docs/.env.example

```
@@ -1,3 +1,7 @@
 NEXT_PUBLIC_POSTHOG_API_KEY=
 NEXT_PUBLIC_POSTHOG_HOST=
-ANTHROPIC_API_KEY= 
\ No newline at end of file
+ANTHROPIC_API_KEY=
+SLACK_WEBHOOK_URL=
+POSTHOG_PERSONAL_API_KEY=
+POSTHOG_PROJECT_ID=
+CRON_SECRET= 
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)